docs/virtual_machines/vms_and_vmis.md

```diff
@@ -0,0 +1,41 @@
+# Virtual Machine types
+There are two main virtual machine user facing API objects: a `VirtualMachine`,
+and a `VirtualMachineInstance`.
+
+## Virtual Machine Instance
+A
+[VirtualMachineInstance](https://kubevirt.io/user-guide/virtual_machines/virtual_machine_instances/)
+represents a single **running** virtual machine.
+
+It is encapsulated in a pod.
+
+When used without a controlling VMI object, it should be limited to running
+stateless workloads.
+
+Removing the VMI object represents stopping the Virtual Machine.
+
+## Virtual Machine
+A [VirtualMachine](http://kubevirt.io/api-reference/main/definitions.html#_v1_virtualmachine)
+in the other hand, represents a VM that is not running or in a stopped state.
```

Comment:
This is very confusing part.

Why does it represent VM that is not running? If I set it to running VirtualMachine no longer represents it?

Reply:
Good point. It's hard to define ... 

At the end of the day, I guess both are true... A running VM is - IMHO - represented by a VMI. Just look at its status object ...

Let me think on how to re-write this...

huge thanks for the feedback!

Reply:
I like to explain / think of this in terms of a Kubernetes Deployment and its Pods. A Deployment can scale to many many Pods, but zero and one are also valid cases. In VMs, we have between zero and two at the most VMIs (migration case) and only one or zero running at a time.

Of course, it's somewhat more complicated with VM -> VMI -> virt-launcher Pod, but I think the analogy helps.